### PR TITLE
Allow URLs to be used as Dropdown items in Tab Widgets #113

### PR DESCRIPTION
**Pull request by Okeanos**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #113

### CODE REVIEW

File: Tabs.php [hunk 1]
```diff
@@ -240,8 +244,11 @@ protected function renderDropdown($itemNumber, &$items, &$panes)
             if (isset($item['visible']) && !$item['visible']) {
                 continue;
             }
-            if (!array_key_exists('content', $item)) {
-                throw new InvalidConfigException("The 'content' option is required.");
+            if (!(array_key_exists('content', $item) xor array_key_exists('url', $item))) {
```
Comment (cebe) [inline Tabs.php]: I'd prefer a `!a && !b` condition here instead of xor :)

Comment (Okeanos) [inline Tabs.php]: I considered that but given that a URL will prevent the content from ever showing up I don't think it is a good idea to actually allow both at the same time from a usability/expected behavior perspective. Using `xor` makes sure that the output is consistent and predictable.

Comment (cebe) [inline Tabs.php]: okay, for better understanding, the error message should be `Either the 'content' or the 'url' option is required, but only one can be set.` then.

**Comment by cebe** [comment]
Thank you!

**Pull request merged**